helpers/notesModel.js

```diff
@@ -0,0 +1,35 @@
+
+//Notes CRUD model
+const db = require('../config/dbConfig')
+
+
+module.exports = {
+
```

Comment:
DB helpers 🔥 These make it much easier to focus on your DB logic and data fetching while avoiding jumbled endpoints that can become hard to debug, good stuff Stefan 👍